src/lib/utils/SynapseClient.ts

```diff
@@ -531,6 +554,13 @@ export const updateEntity = (
   return doPut(url, entity, sessionToken, undefined, endpoint)
 }
 
+export const deleteEntity: GetEntity = <T>(sessionToken: string | undefined = undefined,
```

Comment:
The GetEntity Type could be better named, not sure what though..

Reply:
Agree. I used whatever was there for get. Will rename once we decide:)